src/Microsoft.SqlTools.ServiceLayer/QueryExecution/QueryExecutionService.cs

```diff
@@ -197,45 +197,48 @@ public void InitializeService(ServiceHost serviceHost)
 
             Func<string, Task> queryCreateFailureAction = message => requestContext.SendError(message);
 
-           
-
             ResultOnlyContext<SimpleExecuteResult> newContext = new ResultOnlyContext<SimpleExecuteResult>(requestContext);
 
             // handle sending event back when the query completes
             Query.QueryAsyncEventHandler queryComplete = async q =>
             {
+                Query trashQ;
```

Comment:
Minor: some internal tools like Policheck don't like words like trash. Might consider renaming to removedQuery